### ISSUE TITLE
Video thumbnails not displayed

### ISSUE THREAD

**Issue by Xygyl** [linuxmint/nemo#3418]
### Distribution

Gentoo 2.15

### Package version

6.0.2

### Frequency

Always

### Bug description

There are no video thumbnails despite having (seemingly) completed all necessary steps. I have ffmpegthumbnailer installed, the preview size limit is set to 64GB, totem isn't installed (so there's no conflict in /usr/share/thumbnailers), and ~/.cache/thumbnails doesn't exist.

Thumbnails are displayed on my desktop running AwesomeWM, but they aren't on my laptop running Hyprland. I have assured that both are configured in the exact same way. Furthermore, "nemo --debug" gives the same output on both.

### Steps to reproduce

I'm not sure how to reproduce it; this is the way that it has always been. This has been a problem on my desktop but it's been fixed (I have no idea how).

### Expected behavior

Video thumbnails are displayed. 

### Additional information

I tried looking everywhere but every solution I came across wither wasn't applicable (like ~/.cache/thumbnails not existing) or simply didn't work. I'm not sure if this is the appropriate place to post this, but I'm out of ideas.

**Comment by leigh123linux** [comment]
Whats your cpu count?

https://github.com/linuxmint/nemo/issues/3399#issuecomment-2121486926

**Comment by Xygyl** [comment]
I have a Ryzen 7 3700X

**Comment by Jeremy7701** [comment]
Do you have a file `/usr/share/thumbnailers/ffmpegthumbnailer.thumbnailer` ?
Contents:-
```
[Thumbnailer Entry]
TryExec=ffmpegthumbnailer
Exec=ffmpegthumbnailer -i %i -o %o -s %s -f
MimeType=video/jpeg;video/mp4;video/mpeg;video/quicktime;video/x-ms-asf;video/x-ms-wm;video/x-ms-wmv;video/x-ms-asx;video/x-ms-wmx;video/x-ms-wvx;video/x-msvideo;video/x-flv;video/x-matroska;application/x-matroska;application/mxf;video/3gp;video/3gpp;video/dv;video/divx;video/fli;video/flv;video/mp2t;video/mp4v-es;video/msvideo;video/ogg;video/vivo;video/vnd.divx;video/vnd.mpegurl;video/vnd.rn-realvideo;application/vnd.rn-realmedia;video/vnd.vivo;video/webm;video/x-anim;video/x-avi;video/x-flc;video/x-fli;video/x-flic;video/x-m4v;video/x-mpeg;video/x-mpeg2;video/x-nsv;video/x-ogm+ogg;video/x-theora+ogg;
```

**Comment by Xygyl** [comment]
No, that directory only has `evince.thumbnailer` and `librsvg.thumbnailer`. I have ffmpegthumbnailer installed.

**Comment by Jeremy7701** [comment]
Installing the package should have installed `/usr/share/thumbnailers/ffmpegthumbnailer.thumbnailer`

**Comment by Xygyl** [comment]
Well, it didn't. I tried to make the file with the text you provided and it didn't do anything.

**Comment by Jeremy7701** [comment]
In Mint (Debian edtion) `apt-cache policy ffmpegthumbnailer` shows
```
ffmpegthumbnailer:
  Installed: 2.2.2+git20220218+dfsg-1+b1
  Candidate: 2.2.2+git20220218+dfsg-1+b1
  Version table:
 *** 2.2.2+git20220218+dfsg-1+b1 500
        500 http://ftp.ticklers.org/debian bookworm/main amd64 Packages
```

**Comment by Xygyl** [comment]
I'm on Gentoo, does that make a difference?

**Comment by Jeremy7701** [comment]
Possibly.
See [https://bugs.gentoo.org/833263](https://bugs.gentoo.org/833263)
